knowledge-base/grid-handle-empty-popup-footer.md

```diff
@@ -0,0 +1,460 @@
+---
+title:  Hiding or Removing the Empty Popup Footer when Using the Popup Form Template
```

Comment:
I would shorten this. A / The are not required in headings, and you can add more keywords elsewhere.

```suggestion
title:  Remove Empty Popup Footer when Using Popup Form Template
```